### PR TITLE
docs: refine notification readme

### PR DESCRIPTION
**Pull request by MuyangAmigo**
CLI BREAKING CHANGE:
  

Extension-toolkit BREAKING CHANGE:
 

CLI feat commits:
  

Extension-toolkit feat commits:

### CODE REVIEW

File: templates/bot/js/notification-function-base/README.md [hunk 1]
```diff
@@ -1,43 +1,37 @@
-# Send notifications to Teams
+# Overview of the Notification bot template
 
-The Notification in Teams feature enables you to consume, transform  and post events as plain text or [adaptive cards](https://docs.microsoft.com/microsoftteams/platform/task-modules-and-cards/cards/cards-reference) to an individual, chat, or channel in Teams.
+This template showcases an app that send a message to Teams with Adaptive Cards triggered by a HTTP post request. You can further extend the template to consume, transform and post events to individual, chat or channel in Teams.
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/README.md:3]: It seems the *Adaptive Cards* and *Microsoft Bot Framework* links are removed. Just double confirm is that expected?

File: templates/bot/js/notification-function-base/README.md [hunk 2]
```diff
@@ -1,43 +1,37 @@
...
-  * When your project is deployed to Azure Functions, replace `<endpoint>` with the url from Azure Functions
+## Get Started with the Notification bot
 
 >
 > **Prerequisites**
 >
-> To run locally, you will need:
+> To run the notification bot template in your local dev machine, you will need:
 >
 > - `Node.js` installed locally (recommended version: 14)
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/README.md:14]: I believe now we recommend node 16, so better to update this as well.

File: templates/bot/js/notification-function-base/README.md [hunk 3]
```diff
@@ -46,238 +40,78 @@ This section walks through the generated code. The project folder contains the f
...
+3. [Step 3: Customize where notifications are sent](#step-3-customize-where-notifications-are-sent)
 
-## Add adaptive card actions
-The Adaptive Card action handler feature enables the app to respond to adaptive card actions that triggered by end users to complete a sequential workflow. When user gets an Adaptive Card, it can provide one or more buttons in the card to ask for user's input, do something like calling some APIs, and then send another adaptive card in conversation to response to the card action.
+### Step 1: Customize the trigger point from event source
 
-To add adaptive card actions to notification bot, you can follow the steps [here](https://aka.ms/teamsfx-card-action-response#add-more-card-actions).
+By default Teams Toolkit scaffolds a single `restify` entry point in `src/index.js`. When a HTTP request is sent to this entry point, the default implementation sends a hard-coded Adaptive Card to Teams. You can customize this behavior by customizing `src/index.js`. A typical implementation might make an API call to retrieve some events and/or data, and then send an Adaptive Card as appropriate.
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/README.md:61]: This is the way to add trigger for restify-based notification bot. For azfunc-based one, it's totally another story.

File: templates/bot/js/notification-function-base/README.md [hunk 4]
```diff
@@ -46,238 +40,78 @@ This section walks through the generated code. The project folder contains the f
...
 
-## Add more triggers
+You can also add additional triggers by:
 
-By default, Teams Toolkit scaffolds a single trigger (either a `timer` trigger or a `http` trigger). 
+- Creating new routing: `server.post("/api/new-trigger", ...);`
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/README.md:65]: The same, azfunc-based notification bot is different from restify-based one.

File: templates/bot/js/notification-function-base/README.md [hunk 5]
```diff
@@ -46,238 +40,78 @@ This section walks through the generated code. The project folder contains the f
...
-The notification implementation is in `bot` folder.
-
-The following files provide the business logic for notifications. These files can be updated to fit your business logic requirements. The default implementation provides a starting point to help you get started.
+The following files can be customized and demonstrate an example implementation to get you started.
 
 | File | Contents |
 | - | - |
-| `*Trigger/function.json` | Azure Function bindings for the notification trigger |
-| `src/*Trigger.js` | Notification trigger implementation |
+| `src/index.js` | Application entry point and `restify` handlers for notifications |
```
Comment (swatDong) [inline templates/bot/js/notification-function-base/README.md:47]: Azfunc-based notification bot does not have `restify` handlers.

Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md:47]: Oh yes, messed up when copy paste, I'll update

Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md]: Updated

File: templates/bot/js/notification-function-base/README.md (hunk 2 above)
Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md]: Updated

File: templates/bot/js/notification-function-base/README.md (hunk 3 above)
Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md]: Updated

File: templates/bot/js/notification-function-base/README.md (hunk 4 above)
Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md]: Updated

File: templates/bot/js/notification-function-base/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply templates/bot/js/notification-function-base/README.md]: Yes, was intentional. Keep it more simplified and consistent with other READMEs.

**Review by swatDong — APPROVED** [review]

**Pull request merged**